CMake/FlagsOverride.cmake

```diff
@@ -0,0 +1,7 @@
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
```

Comment:
We could really use some more overrides on other flags / platform, so this should be fine.